bluez-async/README.md

```diff
@@ -0,0 +1,28 @@
+# BlueZ async client
+
+[![crates.io page](https://img.shields.io/crates/v/bluez-async.svg)](https://crates.io/crates/bluez-async)
+[![docs.rs page](https://docs.rs/bluez-async/badge.svg)](https://docs.rs/bluez-async)
+
+`bluez-async` is an async wrapper around the D-Bus interface of BlueZ, the Linux Bluetooth daemon.
+It provides type-safe interfaces to a subset of the Bluetooth client (i.e. central, in Bluetooth
+terminology) interfaces exposed by BlueZ, focussing on the Generic Attribute Profile (GATT) of
+Bluetooth Low Energy (BLE).
+
+See the [examples](examples/) directory for examples of how to use it.
+
+This crate is incomplete, experimental, and may not be supported.
```

Comment:
I was about to suggest `maintenance = { status = "experimental" }` but then I re-read https://doc.rust-lang.org/cargo/reference/manifest.html#the-badges-section and I guess this is just as good.